### PR TITLE
fix qemu driver and builder

### PR DESCRIPTION
**Pull request by vtolstov**

### CODE REVIEW

**Comment by vtolstov** [comment]
fixes #2376

**Comment by vtolstov** [comment]
any news for this pr?

File: builder/qemu/driver.go [hunk 1]
```diff
@@ -188,8 +189,8 @@ func (d *QemuDriver) Version() (string, error) {
 
 	versionOutput := strings.TrimSpace(stdout.String())
 	log.Printf("Qemu --version output: %s", versionOutput)
-	versionRe := regexp.MustCompile("qemu-kvm-[0-9]\\.[0-9]")
-	matches := versionRe.Split(versionOutput, 2)
+	versionRe := regexp.MustCompile("[0-9]\\.[0-9]\\.[0-9]")
```
Comment (imduffy15) [inline builder/qemu/driver.go]: This will not match 0.12.1

**Comment by imduffy15** [comment]
As far as I'm aware when using headless: true on qemu 0.12.1 the argument of -display sdl will through an invalid argument error.

Something like the following might be useful in this PR:

https://github.com/monkeylittleinc/packer/blob/release/builder/qemu/step_run.go#L84

**Comment by vtolstov** [comment]
I'm fix pcre for qemu

**Comment by imduffy15** [comment]
Awesome! Great work @vtolstov 

@cbednarski @mitchellh Can either of you do a review and possible merge? :)

**Comment by markpeek** [comment]
Thanks!

**Pull request merged**